app/scripts/views/home/home-partial.html

```diff
@@ -1,33 +1,23 @@
-<div class="home-page">
+<div class="home-page" ng-class="{'map-expanded': home.mapExpanded}">
     <imls-brand></imls-brand>
 
-    <!-- TODO: Directive -->
-    <div class="map-container">
-        <div id="map"></div>
-    </div> <!-- /.map-container -->
+    <cartodb-vis></cartodb-vis>
 
     <div class="action-bar">
         <div class="container-fluid">
             <form class="form-inline">
                 <div class="action-group">
                     <div class="form-group search-bar">
-                    <div class="input-group">
-                        <label class="input-group-addon" for="searchInput"><i class="md-icon-search"></i></label>
-                        <input type="text" class="form-control search-field" id="searchInput" placeholder="Search">
+                        <div class="input-group">
+                            <label class="input-group-addon" for="searchInput"><i class="md-icon-search"></i></label>
+                            <input type="text" class="form-control search-field" id="searchInput" placeholder="Search">
```

Comment:
input is a void element, as per the spec does not need a closing tag but can be optionally self-closed.

See: http://w3c.github.io/html-reference/syntax.html#void-elements

I didn't modify this one from the prototype, I think the designers typically omit the self-closing `/`